dbt/adapters/databricks/api_client.py

```diff
@@ -356,6 +361,93 @@ def cancel(self, run_id: str) -> None:
         if response.status_code != 200:
             raise DbtRuntimeError(f"Cancel run {run_id} failed.\n {response.content!r}")
 
+    def list_active_runs_for_job(self, job_id):
+        request_body = {
+            'job_id': job_id,
+            'active_only': True,
+        }
+        active_runs_response = self.session.get("/list", json=request_body)
+        if active_runs_response.status_code != 200:
+            raise DbtRuntimeError(f"Error getting active runs.\n {active_runs_response.content!r}")
+
+        return active_runs_response.json().get("runs", [])
+
+class JobPermissionsApi(DatabricksApi):
+    def __init__(self, session: Session, host: str):
+        super().__init__(session, host, "/api/2.0/permissions/jobs")
+
+    def put(self, job_id, access_control_list):
```

Comment:
Should this be part of the operations that dbt handles?  If so, should there be an equivalent for the job runs approach?

Reply:
I'd argue it should be since dbt creates these workflows, so they should also at least allow for the possibility of managing permissions. Otherwise these objects are kind of orphaned and there has to be some other process. Dbt abdicates that responsibility on schemas, which puts everyone in an awkward position.

> If so, should there be an equivalent for the job runs approach?

I'm not sure it's as necessary for job runs, since those are usually just a one-time thing. But I'm happy to add it in there if you think that makes the most sense.

Reply:
Good point about schema.  That's in my backlog of pain points :P

Reply:
I'll take a look at pulling that into job runs too